src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -28,6 +28,9 @@
   <ItemGroup>
     <ProjectCapability Remove="TestContainer" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Particular.CodeRules" Version="0.9.0-token-default.1" />
+  </ItemGroup>
```

Comment:
These (this one and in all the other csprojs) come out eventually because CodeRules is referenced in [Directory.Build.props](https://github.com/Particular/NServiceBus/blob/master/src/Directory.Build.props), right?

Reply:
correct